src/frontend/src/utils/gantt.utils.ts

```diff
@@ -38,6 +38,8 @@ export interface GanttTaskData {
   hideChildren?: boolean;
   displayOrder?: number;
   onClick?: () => void;
+  projectLead?: User;
+  projectManager?: User;
```

Comment:
a bit weird how many props we got here, if u could find a way to group all the props that are just information about the task and not regular react props (e.g. disabled, onClick) into one prop, that would be good

Reply:
also rename isDisabled, to disabled